src/i18n/definitions.js

```diff
@@ -65,6 +65,7 @@ let strings = new LocalizedStrings(
       //CreateAccount
       nameIsRequired: "Name is required.",
       passwordMustBeMsg: "Password should be at least 4 characters long.",
+      passwordMustBeMsg: "You need to agree to our privacy notice.",
```

Comment:
The variable name for the consent validation overlaps with the missing password alert. How about renaming it to `plsAcceptPrivacyPolicy`, for instance?

Reply:
For sure! I was just testing the functionality. It's done!